src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -54,18 +77,70 @@ describe('components/Popups/ReplyField', () => {
         });
     });
 
+    describe('getVirtualElement()', () => {
+        beforeEach(() => {
+            jest.spyOn(window, 'getSelection');
+        });
+
+        test('should return null if no selection', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+
+            let virtualElement = instance.getVirtualElement(mockMention);
+            expect(virtualElement).toBeNull();
+
+            jest.spyOn(window, 'getSelection').mockReturnValue({
+                focusNode: null,
+            } as Selection);
+            virtualElement = instance.getVirtualElement(mockMention);
+
+            expect(virtualElement).toBeNull();
+        });
+
+        test('should return virtual element', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+
+            const mockMentionRect = {
+                x: 0,
+                y: 0,
+            };
+            const mockTextNode = document.createTextNode('');
+            const mockRange = ({
```

Comment:
Can probably just use `document.createRange` and override the two properties?

Reply:
> Can probably just use `document.createRange` and override the two properties?

Also need to override `setStart` and `setEnd` cause they have to be mock to used in `expect`.